pkg/controllers/machine_controller.go

```diff
@@ -173,6 +194,44 @@ func (r *MachineReconciler) Start(ctx context.Context) error {
 	return nil
 }
 
+func (r *MachineReconciler) processVolumeSizeChanges(parent context.Context, log logr.Logger) {
```

Comment:
Can we call this method `startCheckAndEnqueueVolumeResize ` or `startMonitorVolumeForResizeReconcile` - or something like that.